bundles/org.openhab.binding.homekit/src/main/java/org/openhab/binding/homekit/internal/enums/ServiceType.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.homekit.internal.enums;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Enumeration of HomeKit service types.
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ */
+@NonNullByDefault
+public enum ServiceType {
+    // TODO manually check the Homekit specification pdf to ensure all types are covered
+    ACCESSORY_INFORMATION(0x3E, "public.hap.service.accessory-information"),
+    AIR_PURIFIER(0xBB, "public.hap.service.air-purifier"),
+    AUDIO_STREAM_MANAGEMENT(0x127, "public.hap.service.audio-stream-management"),
+    BATTERY(0x96, "public.hap.service.battery"),
+    CAMERA_RTP_STREAM_MANAGEMENT(0x110, "public.hap.service.camera-rtp-stream-management"),
+    DATA_STREAM_TRANSPORT_MANAGEMENT(0x129, "public.hap.service.data-stream-transport-management"),
+    DOOR(0x81, "public.hap.service.door"),
+    DOORBELL(0x121, "public.hap.service.doorbell"),
+    FANV2(0xB7, "public.hap.service.fanv2"),
```

Comment:
FAN(0x40, "public.hap.service.fan")

still in use by many accessories, and only documented in older versions of the non-commercial HAP specification